templates/customize-adventure.html

```diff
@@ -47,7 +47,7 @@ <h3 class="text-center mt-0 mb-4">{{ _('adventure') }}</h3>
                         <label for="agree_public" class="text-sm italic w-full">{{_('adventure_terms')}}</label>
                     </div>
                     {% if class_data %}
-                        <div class="flex flex-col items-center">
+                        <!-- <div class="flex flex-col items-center">
```

Comment:
This is commented out. Is this left in for debugging purposes?

If this was intentional, then wouldn't we better delete the code?

(Approving this PR regardless since it is a bugfix and it is important to get out).

Reply:
It's a feature that I plan to reenable today!